formats/sarifutils/test_sarifutils.go

```diff
@@ -13,14 +13,16 @@ func CreateRunWithDummyResults(results ...*sarif.Result) *sarif.Run {
 	return run
 }
 
-func CreateRunWithDummyResultAndRuleProperties(property, value string, result *sarif.Result) *sarif.Run {
+func CreateRunWithDummyResultAndRuleMultipleProperties(result *sarif.Result, properties, values []string) *sarif.Run {
```

Comment:
Also - why did you switch the order of the args?

Reply:
I thought it was nicer to have the arrays at the end of the function, what do you think?